docs/source/_toctree.yml

```diff
@@ -63,6 +63,20 @@
     title: 'Performance and Scalability: How To Fit a Bigger Model and Train It Faster'
   - local: parallelism
     title: Model Parallelism
+  - local: perf_infer
```

Comment:
Let's make a subfolder instead of having so many document names prefixed with `perf`.

Reply:
great idea